Pipfile

```diff
@@ -16,6 +17,7 @@ pytest-xdist = "*"
 pyflakes = "*"
 tag-version = "*"
 responses = "*"
+protobuf = ">=3.20.2"
```

Comment:
Do we need both?

Reply:
![Screenshot 2022-11-03 at 10 18 12 AM](https://user-images.githubusercontent.com/31461850/199790433-90615ba4-e784-4220-8a38-e12def5cf936.png)